tests/unit/models/proposal-test.js

```diff
@@ -1,49 +1,8 @@
-import { moduleFor, test } from 'ember-qunit';
-import schemas from 'npm:kosmos-schemas';
-import tv4 from 'npm:tv4';
-
-moduleFor('model:proposal', 'Unit | Model | proposal');
-
-test('#toJSON() requires a recipient profile IPFS hash to be set', function(assert) {
-  let model = this.subject();
-
-  model.setProperties({
-    recipientAddress: '0xd4a64570b12da659ee4bbd41c3509b7b1f9c51ac'
-  });
-
-  try {
-    let json = model.contributionToJSON();
-    assert.notOk(json, true);
-  } catch(e) {
-    assert.ok(e.message.match(/IPFS hash .* missing/i));
-  }
-});
-
-test('#toJSON() requires kind and description to be set', function(assert) {
-  let model = this.subject();
-
-  model.setProperties({
-    recipientProfile: 'QmT2A7rY4e7uoKktkcFHQNN7BD1oXdZTgd8wNkr1u9nNVE'
-  });
-
-  try {
-    let json = model.contributionToJSON();
-    assert.notOk(json, true);
-  } catch(e) {
-    assert.ok(e.message.match(/Missing .* kind.*description/i));
-  }
-});
```

Comment:
The validations seem to be missing in the new serializer tests.

Reply:
Yes. I'll add it in `addContributor` and `addProposal` and use `v4` to do it. I think it's cleaner to validate before we save instead of while we serialize.

Reply:
Sounds good.